.github/workflows/pytorch.yml

```diff
@@ -33,16 +33,13 @@ jobs:
       - uses: bytedeco/javacpp-presets/.github/actions/deploy-ubuntu@actions
         timeout-minutes: 350
   macosx-arm64:
-    runs-on: macos-12
+    runs-on: macos-14
     steps:
-      - uses: bytedeco/javacpp-presets/.github/actions/deploy-macosx@actions
+      - uses: HGuillemet/javacpp-presets/.github/actions/deploy-macosx@hg_pytorch
   macosx-x86_64:
-    runs-on: macos-12
-#    strategy:
-#      matrix:
-#        ext: ["", -gpu]
+    runs-on: macos-13
```

Comment:
Let's leave that to 12

Reply:
I had errors compiling Pytorch 2.4.0 with Apple Clang from macos-12 runner.
What is the problem with using the macos-13 runner ?

Reply:
The problem is that it doesn't work either, right? I don't think the issue is with the version of macOS

Reply:
With macos-13, there was a little issue because, apparently, new versions of MacOS 13 provide Python 12 with symlink in /usr/local. And  Gradle from brew has a dependency towards Python 12, and triggers the creation of symlinks, clashing with the one already existing. It works by installing gradle with `--ignore-dependencies`. macos-12 runner also provides python3.12, so I think the workflows using macos-12 and needing Python or Gradle will still work but this will require verification. Or we can use macos-13 runner everywhere.

Pytorch compilation error on macos-12 is completly different. Something related to c++17 (`std::unordered_map`) support of the included Clang. See https://github.com/bytedeco/javacpp-presets/actions/runs/10197815123/job/28211441407.

I won't mind dropping macosx-x86_64 platform for Pytorch if this cannot be solved easily. I doubt many people will try to run Pytorch for Java on an Intel Mac nowadays.

Reply:
It just sounds like `#include <unordered_map>`  and `#include <vector>` are missing from Handlers.cpp, let's just do that instead?